ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md

```diff
@@ -1,41 +1,21 @@
 ---
-name: Proposal Template
+name: 💭 Proposal
 about: Suggest an idea for a specific feature you wish to propose to the community for comment
-title: ''
+title: '[PROPOSAL]'
 labels: proposal
 assignees: ''
 ---
+## What kind of business use case are you trying to solve? What are your requirements?
+A clear and concise description of the business problem, e.g. _My customers are asking for [...]_
 
-<!--
-Welcome to the OpenSearch project! 👋🎉
+**What is the problem? What is preventing you from meeting the requirements?**
+Describe what impact this problem has for lack of a solution.
 
+**What are you proposing? What do you suggest we do to solve the problem or improve the existing situation?**
+Describe your proposed solution. It's OK if you don't have one.
 
- - Please be respectful and considerate of others when commenting on issues.
- - Search for existing issues before creating duplicate proposals
- - Try to provide as much detail as possible
- - Please delete the comments before posting 
- - When opening a feature request you should expect to contribute a solution or part of this to make it a reality. 
- - Think of this like a requirements doc a product manager may fill out but with less rigor typically
```

Comment:
This is some content from the COC, and some that's general across all issues, so I removed it to avoid saying different things in different places. The comments are also not something GitHub shows in any interesting ways, so I think writers of issues should overwrite the text just like in the bug and feature templates alongside proposal.

Now, if we do think we should have a comment section like this, we can add it in all templates. In this PR I am voting to remove it.